test/parallel/test-process-hrtime.js

```diff
@@ -24,3 +24,9 @@ function validateTuple(tuple) {
     assert(isFinite(v));
   });
 }
+
+const base = process.hrtime();
+for (let i = 0; i < 1e5; i += 1) {
```

Comment:
I think using `assert(process.hrtime([0, 999999999])[1] >= 0);` as @ChALkeR suggested would be better than my test.